bin/circleci/setup-integration-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+
+# TODO: Update this occasionally from here:
```

Comment:
Might also be nice to do boring stuff like update pip/npm dependencies on a twice-monthly schedule. Thoughts?